pkg/server/event_handler.go

```diff
@@ -341,26 +341,44 @@ type notificationPayload struct {
 }
 
 func parseGitHubSource(ctx context.Context, data []byte, objAttrs map[string]string) (*v1alpha1.GitHubSource, error) {
+	logger := logging.FromContext(ctx)
+
 	var pm *notificationPayload
 	if err := json.Unmarshal(data, &pm); err != nil {
 		return nil, fmt.Errorf("failed to unmarshal payloadMetadata %w", err)
 	}
 
 	var r v1alpha1.GitHubSource
 
-	if c, found := pm.Metadata[MetadataKeyGitHubCommit]; found {
+	// Set github-commit.
+	c, found := pm.Metadata[MetadataKeyGitHubCommit]
+	if !found {
+		logger.Infof("github-commit not found")
```

Comment:
nits: Just use the const defined in the file `MetadataKeyGitHubCommit` instead of repeat the raw string.

Reply:
Updated & marked as resolved.